nautobot/utilities/utils.py

```diff
@@ -340,6 +340,8 @@ def to_meters(length, unit):
         return length * Decimal("0.3048")
     if unit == CableLengthUnitChoices.UNIT_INCH:
         return length * Decimal("0.3048") * 12
+    if unit == CableLengthUnitChoices.UNIT_MILE:
```

Comment:
Don't we also need to add a case for `CableLengthUnitChoices.UNIT_KILOMETER`?

Reply:
ah shoot, you're right. I totally missed that.

It's now been added.